src/Microsoft.Win32.Registry/src/PinvokeAnalyzerExceptionList.analyzerdata

```diff
@@ -0,0 +1,21 @@
+advapi32.dll!RegEnumValueW
```

Comment:
you can do that by setting `<EnableDotnetAnalyzers>false</EnableDotnetAnalyzers>` on each of the projects.